app/src/main/java/e/josephmolina/saywhat/Modules/SayWhatApp.java

```diff
@@ -0,0 +1,19 @@
+package e.josephmolina.saywhat.Modules;
+
+import android.app.Application;
+
+import com.squareup.leakcanary.LeakCanary;
+
+public class SayWhatApp extends Application {
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        if (LeakCanary.isInAnalyzerProcess(this)) {
+            // This process is dedicated to LeakCanary for heap analysis.
+            // You should not init your app in this process.
```

Comment:
Can you change this comment to  `You should not init the app in this process.`? If this was a tutorial it would make sense to say that to a developer adding a snippet of code to their app, but here it is a reminder to other developers and other developers may not be the owner of the app.